### PR TITLE
test downstream projects

### LINKED ISSUE #44: Add CI job for downstream projects
Following https://github.com/inducer/meshmode/pull/114

Probably should happen at the same time as https://github.com/inducer/meshmode/issues/115

@matthiasdiener Could you prepare a PR along these lines?

### PR DESCRIPTION
**Pull request by matthiasdiener**
Fixes #44.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,6 +65,44 @@ jobs:
...
-# vim: sw=4
+    downstream_tests:
+        strategy:
+            matrix:
+                # Reinstate when
+                # https://github.com/illinois-ceesd/mirgecom/issues/212
+                # is fixed:
+                # downstream_project: [meshmode, pytential, pytato, mirgecom]
 
+                downstream_project: [meshmode, pytential, pytato]
```
Comment (inducer) [inline .github/workflows/ci.yml:76]: meshmode and pytential aren't downstream of grudge?

Comment (matthiasdiener) [reply .github/workflows/ci.yml:76]: Ok, but which projects would you like to see tested here? I'm not sure that pytato is downstream of grudge either.

Comment (inducer) [reply .github/workflows/ci.yml:76]: Only mirgecom is downstream of grudge.

**Comment by inducer** [comment]
Unsubscribing... @-mention or request review once it's ready for a look or needs attention.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (matthiasdiener) [reply .github/workflows/ci.yml:76]: Ok, so does it make sense to merge this PR, considering that illinois-ceesd/mirgecom#212 is still open?

**Comment by matthiasdiener** [comment]
Current error is: 

```
+++ python3 -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw -n 4 .
============================= test session starts ==============================
platform linux -- Python 3.8.8, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/runner/work/grudge/grudge/mirgecom, configfile: pytest.ini
plugins: forked-1.3.0, xdist-2.2.1
gw0 I / gw1 I / gw2 I / gw3 I
gw0 [167] / gw1 [167] / gw2 [167] / gw3 [167]

........................................................................ [ 43%]
..............................[gw1] node down: Not properly terminated
F
replacing crashed worker gw1
..........................................[gw0] node down: Not properly terminated
F
replacing crashed worker gw0
.....................
=================================== FAILURES ===================================
______________________________ test/test_euler.py ______________________________
[gw1] linux -- Python 3.8.8 /home/runner/work/grudge/grudge/mirgecom/.miniforge3/envs/testing/bin/python3
worker 'gw1' crashed while running "test/test_euler.py::test_multilump_rhs[<array context factory for <pyopencl.Device 'pthread-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz' on 'Portable Computing Language'>-1.0-4-3]"
______________________________ test/test_wave.py _______________________________
[gw0] linux -- Python 3.8.8 /home/runner/work/grudge/grudge/mirgecom/.miniforge3/envs/testing/bin/python3
worker 'gw0' crashed while running "test/test_wave.py::test_wave_stability[<array context factory for <pyopencl.Device 'pthread-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz' on 'Portable Computing Language'>-problem3-0.025-2]"
=============================== warnings summary ===============================
```

**Comment by inducer** [comment]
Thx!

**Pull request merged**